_includes/sections/dns.html

```diff
@@ -285,6 +285,22 @@ <h4>Terms</h4>
     <li>DNSCrypt - An older yet robust method of encrypting DNS.</li>
   </ul>
 
+  <h4>How to Validate</h4>
+
+  <ul>
+    <li>DoH / DoT
+      <ul>
+        <li>Check <a href="https://www.dnsleaktest.com/">https://www.dnsleaktest.com/</a>.</li>
+        <li>Check the website of your DNS provider. They may have a page for telling "you are using X DNS." Examples include <a href="https://adguard.com/en/adguard-dns/overview.html">AdGuard</a> and <a href="https://1.1.1.1/help">Cloudflare</a>.</li>
+        <li>If using Firefox's trusted recursive resolver (TRR), navigate to <code>about:networking#dns</code>. If the TRR column says "true" for some fields, you are using DoH.</li>
+      </ul>
+    </li>
+    <li>dnscrypt-proxy - Try the above steps or attempt to stop it. If you have configured it correctly, your DNS requests will stop working (with the exception having Firefox's TRR configured).
+    </li>
+    <li>DNSSEC - Check <a href="https://dnssec.vs.uni-due.de/">https://dnssec.vs.uni-due.de/</a>.</li>
```

Comment:
```suggestion
    <li>DNSSEC - Check <a href="https://dnssec.vs.uni-due.de/">dnssec.vs.uni-due.de</a>.</li>
```

I want the link to look better, but I am not sure what to call that test as they don't call themselves as "DNSSEC capability checker by ??? University" or similar.

Reply:
yeah :thinking:

Reply:
I would literally just call it "DNSSEC Resolver Test by University of Duisburg-Essen"

Reply:
Alternatively, "DNSSEC Resolver Test by Matthäus Wander"